web/themes/custom/votegov/templates/navigation/menu--main--votegov-mainnavigation.html.twig

```diff
@@ -36,6 +36,12 @@ We call a macro which calls itself to render the full tree.
   <div class="usa-nav__secondary">
     {{ drupal_entity('block_content', 2) }}
   </div>
+  {% if language == 'en' %}
+      <div class="usa-nav__secondary">
```

Comment:
i dont think we need this div of nav__ secondary. when i removed the div it fixed the spacing issue